content/en/docs/components/model-registry/installation.md

```diff
@@ -31,13 +31,13 @@ Remember to substitute the relevant release (e.g. `v0.1.2`), modify `ref=main` t
 The following steps show how to install Model Registry using a default Kubeflow >=1.8 installation.
 
 ```shell
-kubectl apply -k "https://github.com/kubeflow/model-registry/tree/main/manifests/kustomize/overlays/db?ref=main"
+kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/overlays/db?ref=main"
 ```
 
 As the default installation provides an Istio mesh, apply the necessary manifests:
 
 ```shell
-kubectl apply -k "https://github.com/kubeflow/model-registry/tree/main/manifests/kustomize/options/istio?ref=main"
+kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=main"
```

Comment:
Some CRDs are missing.

```bash
➜  ~ kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=main"
resource mapping not found for name: "model-registry-service" namespace: "kubeflow" from "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=main": no matches for kind "DestinationRule" in version "networking.istio.io/v1alpha3"
ensure CRDs are installed first
resource mapping not found for name: "model-registry" namespace: "kubeflow" from "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=main": no matches for kind "VirtualService" in version "networking.istio.io/v1alpha3"
ensure CRDs are installed first
resource mapping not found for name: "model-registry-service" namespace: "kubeflow" from "https://github.com/kubeflow/model-registry/manifests/kustomize/options/istio?ref=main": no matches for kind "AuthorizationPolicy" in version "security.istio.io/v1beta1"
ensure CRDs are installed first
```

Reply:
Based on the error message, it looks to me you are trying to apply the Istio-related manifests, in an Kubeflow installation in a cluster without Istio 🤔 Which steps have you followed to install Kubeflow, so I could try to replicate your errror, please?

Reply:
Oh, I didn't install Kubeflow. Now re-reading the installation instructions I see that those commands require it.

> If you want to install Model Registry separately from Kubeflow

This led me to think that Kubeflow is not a requirement.

Reply:
fwiw the instructions work with a vanilla KF:
![image](https://github.com/kubeflow/website/assets/1699252/a901b8d3-41a1-4cee-9ff9-5e77b9e04a7c)

but fair point about the potential confusion of the wording, will amend 👍

Reply:
done with https://github.com/kubeflow/website/pull/3698/commits/ba73b17415cd273d27771a50be52604a98308273